spyder/utils/misc.py

```diff
@@ -15,6 +15,7 @@
 import stat
 import socket
 
+from qtpy.QtCore import QFile
```

Comment:
```suggestion
from qtpy.QtCore import QFile

```

Our custom is to add third-party imports to a different section from local imports.